src/modules/editor/ScriptEditorImplementation.cpp

```diff
@@ -67,6 +67,7 @@ extern std::set<ScriptEditorImplementation *> g_pScriptEditorWindowList;
 extern KviModule * g_pEditorModulePointer;
 
 static QColor g_clrBackground(0, 0, 0);
+static QColor g_clrBackgroundHighlight(50, 50, 50);
```

Comment:
Im not sure  #323232 Is that great now I see it implemented. It works as desired, but this means users have to select a dark color for highlight which feels counter intuitive (if user prefers a light theme the line highlight must be light as opposed to dark also counter intuitive).

Im not sure what a better alternative would be, but usually this is done with a transparency e.g.
`static QColor g_clrSomeTarget(198, 198, 198, .4);`

not sure how this would work for user set colors considering....

maybe we should use tooltips here too....

I found a bug though. when changing the editor colors to a light theme most colors were not applied until an editor restart. see scrrenshot

![capture](https://cloud.githubusercontent.com/assets/3521959/17830349/f85c3818-66c0-11e6-9bde-89e876512967.PNG)